src/sources/http_server.rs

```diff
@@ -1115,4 +1227,66 @@ mod tests {
 
         assert_eq!(200, send_request(addr, "GET", "", "/").await);
     }
+
+    #[test]
+    fn output_schema_definition_vector_namespace() {
+        let config = SimpleHttpConfig {
+            log_namespace: Some(true),
+            ..Default::default()
+        };
+
+        let definition = config.outputs(LogNamespace::Vector)[0]
+            .clone()
+            .log_schema_definition
+            .unwrap();
+
+        let expected_definition =
+            Definition::new_with_default_metadata(Kind::bytes(), [LogNamespace::Vector])
+                .with_meaning(LookupBuf::root(), "message")
+                .with_metadata_field(&owned_value_path!("vector", "source_type"), Kind::bytes())
+                .with_metadata_field(
+                    &owned_value_path!(SimpleHttpConfig::NAME, "path"),
+                    Kind::bytes(),
+                )
+                .with_metadata_field(
+                    &owned_value_path!(SimpleHttpConfig::NAME, "headers"),
+                    Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                )
+                .with_metadata_field(
+                    &owned_value_path!(SimpleHttpConfig::NAME, "query_parameters"),
+                    Kind::object(Collection::empty().with_unknown(Kind::bytes())).or_undefined(),
+                )
+                .with_metadata_field(
+                    &owned_value_path!("vector", "ingest_timestamp"),
+                    Kind::timestamp(),
+                );
```

Comment:
I _think_ we want to define the schema in the same order in which the fields are inserted.

Reply:
Oh.. that is news :+1:

Reply:
:thinking: why does the order matter? Wondering if we should test that too...

Reply:
iirc the ordering doesn't matter anymore.